android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy

```diff
@@ -44,7 +44,7 @@ class AvdCreator {
         avdStore.store(avdName, settings) {
             def avdmanager = new CommandBuilder(Utils.joinPaths(sdkDirectory, 'tools', 'bin', 'avdmanager'), '.bat')
 
-            avdmanager.addArguments(['create', 'avd', '-f', '-n', avdName])
+            avdmanager.addArguments(['create', 'avd', '-f', '-verbose', '-n', avdName])
```

Comment:
That did not work for me.
It mentions that there is no "-verbose" flag,  instead there is "-v" and "--verbose".
Yet neither of these worked and I got "Error: Flag '--verbose' is not valid for 'create avd'."

Did this work for you?
Maybe we have different versions of the Android SDK.

Reply:
Hi, no we decided to not merge this pull request, I corrected it an will test it today.
Thanks